packages/shell-api/src/helpers.ts

```diff
@@ -614,18 +614,30 @@ export async function getPrintableShardStatus(
               );
             }
 
-            const tagsRes: any[] = [];
-            for await (const tag of (
+            const tags = await (
               await configDB.getCollection('tags').find({
                 ns: coll._id,
               })
-            ).sort({ min: 1 })) {
-              tagsRes.push({
-                tag: tag.tag,
-                min: tag.min,
-                max: tag.max,
-              });
+            )
+              .sort({ min: 1 })
+              .toArray();
+            const tagsRes: any[] = [];
+
+            // NOTE: this will return tags as a string, and will print ugly BSON
```

Comment:
I'm not quite sure I understand this comment – the tags are still regular tags, right?